### PR TITLE
CORE-39919 Add information about how to create and update adobe.ly URLs.

### PR DESCRIPTION
**Pull request by Aaronius**
## Description
Add information about how to create and update adobe.ly URLs.

## Related Issue
https://jira.corp.adobe.com/browse/CORE-39919

## Motivation and Context
Inform users on how to make and change adobe.ly URLs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [x] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,13 @@ For functional testing, please see the [functional testing documentation](test/d
 
 When you attempt to commit code changes, several of the above tasks will be run automatically to help ensure that your changes pass tests and are consistent with agreed-upon standards.
 
-Thank you for your interest in contributing! 
+Thank you for your interest in contributing!
+
+## Linking to Documentation
+
+If you are writing code that logs a message referring the user to a particular piece of [documentation](https://app.gitbook.com/@launch/s/adobe-experience-platform-web-sdk), please link to the documentation using an `adobe.ly` URL. To make an `adobe.ly` URL, go to [Bitly](https://bitly.com/) and shorten the documentation URL. Bitly should provide you with a short url using the `adobe.ly` domain.
```
Comment (liljenback) [inline README.md]: Do you mean bit.ly here when saying adobe.ly?

Comment (Aaronius) [reply README.md]: I did mean bit.ly. Am I missing something? As far as I'm aware, I'm unable to make an adobe.ly URL by going to adobe.ly.

Comment (Aaronius) [reply README.md]: Oh wait. I read your comment wrong. Yes, I meant adobe.ly when I said adobe.ly. Is it wrong for some reason?

**Comment by Aaronius** [comment]
Done. Thanks @liljenback.

**Pull request merged**